jenkins/ring-install.sh

```diff
@@ -278,8 +278,12 @@ function _configure_sproxyd {
     sudo sed -i '/by_path_cos/d;/by_path_service_id/d' /etc/sproxyd.conf
     sudo sed -i '/ring_driver:0/a\        "by_path_cos": 0,' /etc/sproxyd.conf
     sudo sed -i '/ring_driver:0/a\        "by_path_service_id": "0xC0",' /etc/sproxyd.conf
+
+    sudo sed -i '/ring_driver:1/a\        "by_path_cos": 1,' /etc/sproxyd.conf
+    sudo sed -i '/ring_driver:1/a\        "by_path_service_id": "0xC1",' /etc/sproxyd.conf
+
     # The next line needs the Chord ring driver to be defined first, ie before the Arc ring driver.
-    sudo sed -i '0,/"by_path_enabled": / { s/"by_path_enabled": false/"by_path_enabled": true/ }' /etc/sproxyd.conf
+    sudo sed -i '/"by_path_enabled": / { s/"by_path_enabled": false/"by_path_enabled": true/ }' /etc/sproxyd.conf
```

Comment:
The code comment L285 doesn't hold anymore with this change. Because both ring driver use are now "by path" the order in which they are defined doesn't matter. I suggest you remove this not-true-anymore code comment.